clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThread.java

```diff
@@ -51,153 +40,47 @@
  * It holds a reference to the {@link SubscriptionState}, which is
  * initialized by the polling thread.
  */
-public class DefaultBackgroundThread extends KafkaThread {
+public class DefaultBackgroundThread<K, V> implements Runnable, Closeable {
```

Comment:
I guess we want this class parametrized for an upcoming change, but just checking if this is intentional, as it seems unused for now

Reply:
Good catch. I'll double check.